incubator/jaeger/templates/collector-deploy.yaml

```diff
@@ -0,0 +1,45 @@
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: "{{ template "jaeger.fullname" . }}-collector"
+  labels:
+    app: "{{ template "jaeger" . }}"
+    jaeger-infra: collector-deployment
+    chart: {{ .Chart.Name }}-{{ .Chart.Version | replace "+" "_" }}
+    component: "{{ template "jaeger.fullname" . }}-collector"
+    heritage: "{{ .Release.Service }}"
+    release: "{{ .Release.Name }}"
+{{- if .Values.deployment.collector.annotations }}
+  annotations:
+{{ toYaml .Values.deployment.collector.annotations | indent 6 }}
+{{- end }}
+spec:
+  replicas: 1
+  strategy:
+    type: Recreate
+  template:
+    metadata:
+      labels:
+        jaeger-infra: collector-pod
```

Comment:
`app`, `component`, and `release` labels should go here.

Reply:
Added app, component, and release.